sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/isa.yaml

```diff
@@ -0,0 +1,60 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+Device: rv32imc
+Vendor: opentitan
+
+ISA: RV32IMC
+misa:
+  implemented: True
+  MXL:
+    range:
+      rangelist: [[1]]
+      mode: Unchanged
+  Extensions:
+    bitmask:
+      mask: 0x0
+      default: 0x1104
+hw_data_misaligned_support: True
+mtvec:
+  MODE:
+    range:
+      rangelist: [[1]]
+  BASE:
+    range:
+      rangelist: [[0x20000020]]
+
+mstatus:
+  MPP:
+    range:
+      rangelist: [[3]]
+
+User_Spec_Version: "2.3"
+Privilege_Spec_Version: "1.11"
+
+mvendorid:
+  implemented: false
+marchid:
+  implemented: false
+mimpid:
+  implemented: false
+mhartid: 0
+
+mcycle:
+  is_hardwired: true
+  implemented: true
+minstret:
+  is_hardwired: true
+  implemented: true
+
+## dont know how to put these in the yaml
+# --override riscvOVPsim/cpu/tvec_align=0
+
+## and then we really need to be able to put in these in the config file...:
+# --override riscvOVPsim/cpu/addrbits=32
+# --override riscvOVPsim/cpu/wfi_is_nop=F
+# --override riscvOVPsim/cpu/tval_ii_code=T
+
+## questions
+## if misa Extensions bitmask mask is left out... the yaml defaults to zero... ovpsim does not
```

Comment:
We don't use OVPsim, so I think these can all be dropped.

Reply:
removed